tasks/eth/011-proto-ver+ownership-transfers/README.md

```diff
@@ -0,0 +1,34 @@
+# Mainnet Recommended Protocol Version Update (7.0.0 - Fjord) & Ownership Transfers
+
+Status: DRAFT
```

Comment:
per the list of [allowed statuses](https://github.com/ethereum-optimism/superchain-ops/blob/fc2bf8c5dff3a7643c5159565e2e1fa23e5789cc/script/utils/check-task-statuses.sh#L4)—this will cause CI to fail once this is updated against the latest main
```suggestion
Status: DRAFT, NOT READY TO SIGN
```